checklist.md

```diff
@@ -0,0 +1,20 @@
+# Checklists
+
+## Suspending Someone
+
+1. Suspend on GitHub
+2. Suspend on Matrix with mjolnir
+3. Suspend on Discourse
+4. Remove inappropriate content
+5. Add entry to moderation-log.md
+  - Include the length of the suspension, if not indefinite
+  - Do __not__ include why the suspension was issued
+
+### Indefinite Suspensions
+
```

Comment:
Same, send an email. Even if the person is behaving badly, we should still treat them with respect (even if it's really hard).